source/speech/speech.py

```diff
@@ -2717,7 +2718,7 @@ def getFormatFieldSpeech(  # noqa: C901
 				# Translators: Reported when text is no longer marked as emphasised 
 				else _("not emphasised"))
 			textList.append(text)
-	if  formatConfig["reportFontAttributes"]:
+	if formatConfig["fontAttributeReporting"] & OutputMode.SPEECH:
```

Comment:
Idem:

```suggestion
	if formatConfig["fontAttributeReporting"] in (OutputMode.SPEECH, OutputMode.SPEECH_AND_BRAILLE):
```

Reply:
>This is especially relevant here since there is nothing explicit in OutputMode enum definition indicating that it should honour such bitwise behaviour.

@CyrilleB79 - I've made a suggestion here to make this more explicit: https://github.com/nvaccess/nvda/pull/16748#discussion_r1665176647

Reply:
Of course, we can document that the enum is meant to be used with bitwise operations if that's what we really want.

But as expressed before, bitwise operations are not used in NVDA's high level control code.

For now, I cannot find the PR where I have been asked to remove bitwise operations from high level control code to illustrate this, sorry. But I remember that the following was considered more explicit and more readable:
`if configValue in (option1, option2):`